dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/StaticLinkageCheckOptionTest.java

```diff
@@ -30,27 +30,29 @@ public void parseCommandLineOptions_shortOptions_bom() throws ParseException {
     };
     StaticLinkageCheckOption parsedOption = StaticLinkageCheckOption.parseArguments(arguments);
 
-    Assert.assertEquals("abc.com:dummy:1.2", parsedOption.getBomCoordinates());
+    Assert.assertEquals("abc.com:dummy:1.2", parsedOption.getBom());
+    Assert.assertTrue(parsedOption.getArtifacts().isEmpty());
```

Comment:
`Truth.assertThat(parsedOption.getArtifacts()).isEmpty()` will give you a better error message on failure.

Reply:
Changed to use Truth.